pipeline/extract.py

```diff
@@ -132,6 +126,22 @@ def scrape_from_debenhams_html(html_content: bytes, url: str, product_id: int) -
     original_price = original_price_element.text.strip(
     ) if original_price_element else current_price
 
+# product description and images
```

Comment:
Does this comment need to be included? It's relatively clear what the code does without it